kotest-framework/kotest-framework-multiplatform-plugin-gradle/build.gradle.kts

```diff
@@ -99,20 +103,20 @@ gradlePlugin {
 }
 
 
-val kotestPluginConstantsFileContents = resources.text.fromString(
-   """
-      |// Generated file, do not edit manually
-      |@file:org.gradle.api.Generated
-      |
-      |package io.kotest.framework.multiplatform.gradle
-      |
-      |const val KOTEST_COMPILER_PLUGIN_VERSION: String = "${Ci.gradleVersion}"
-      |
-   """.trimMargin()
-)
-
 val updateKotestPluginConstants by tasks.registering(Sync::class) {
 
+   val kotestPluginConstantsFileContents: TextResource = resources.text.fromString(
+      """
+         |// Generated file, do not edit manually
+         |@file:org.gradle.api.Generated
+         |
+         |package io.kotest.framework.multiplatform.gradle
+         |
+         |const val KOTEST_COMPILER_PLUGIN_VERSION: String = "${Ci.gradleVersion}"
+         |
+      """.trimMargin()
+   )
```

Comment:
I moved this to avoid an issue with the Gradle configuration cache, which likes having variables defined within the task.